### PR TITLE
Remove heavy SVG in favor of Webp

### PR DESCRIPTION
**Pull request by FedeMadoery**
- [x] Add .webp images instead of heavy svg
- [x] Replace adornments svgs
- [x] Add resize for adornment images
- [x] Move HowTo ilustrations to pure .svg (now we have texts for SEO and less file size)

### New build log
```
Page                                       Size     First Load JS
┌ ○ / (1631 ms)                            720 kB          839 kB
├   /_app                                  0 B             118 kB
└ ○ /404                                   186 B           119 kB
+ First Load JS shared by all              118 kB
  ├ chunks/framework-9b5d6ec4444c80fa.js   45.7 kB
  ├ chunks/main-f0e16f48d3775e5e.js        30.7 kB
  ├ chunks/pages/_app-b08dae46b209c1fb.js  40.2 kB
  └ chunks/webpack-2e68b01c7e74ed91.js     1.83 kB

○  (Static)  automatically rendered as static HTML (uses no initial props)
```

Previos build log

```
Page                                       Size     First Load JS
┌ ○ / (1943 ms)                            4.45 MB        4.56 MB
├   /_app                                  0 B             118 kB
└ ○ /404                                   186 B           118 kB
+ First Load JS shared by all              118 kB
  ├ chunks/framework-9b5d6ec4444c80fa.js   45.7 kB
  ├ chunks/main-f0e16f48d3775e5e.js        30.7 kB
  ├ chunks/pages/_app-b08dae46b209c1fb.js  40.2 kB
  └ chunks/webpack-71eb4fcc773b358e.js     1 kB

○  (Static)  automatically rendered as static HTML (uses no initial props)
```

### CODE REVIEW

File: src/components/CardType/index.tsx [hunk 1]
```diff
@@ -22,28 +21,25 @@ type BadgeTypeDescriptionPropsType = {
...
+  imageSizes,
   adornment,
-  howToImage,
-  howToImageAlt,
-  howToImageAspectRatio,
+  howToSVGComponent,
+  howToAspectRatio,
 }: BadgeTypeDescriptionPropsType) => {
-  const isMobile = useIsMobile();
+  // const HowTo: ComponentType<SvgIconProps> | null = howToSVGComponent;
```
Comment (Agupane) [inline src/components/CardType/index.tsx:42]: is this needed?

File: src/components/CardType/ThirdPartyCard.tsx [hunk 2]
```diff
@@ -16,15 +27,24 @@ export const ThirdPartyCard = () => {
...
-      Step 1: First the entity has to register within the platform submitting information such as name, description, logo, etc.
-      Step 2: Once they are registered, they are able to mint their own custom badges according to their needs.
-      Step 3: In case they want to, they might apply as verified entities by The Badge.
-      Step 4: If they get verified, they will be able not only to whitelist addresses but also to directly mint badges for their users.
-      `}
+      image={image_desktop}
+      imageSizes={
+        isMobile
+          ? {
+              width: 96,
```
Comment (Agupane) [inline src/components/CardType/ThirdPartyCard.tsx:34]: can be this on the theme?

File: src/components/CardType/ThirdPartyCard.tsx [hunk 3]
```diff
@@ -16,15 +27,24 @@ export const ThirdPartyCard = () => {
...
+              width: 210,
+              height: 300,
+            }
+      }
+      howToSVGComponent={
+        <Suspense fallback={<CircularProgress color="success" />}>
+          {isMobile ? <ThirdPartyHowToMobile /> : <ThirdPartyHowTo />}
+        </Suspense>
+      }
+      howToAspectRatio={isMobile ? '6/7' : '10/9'}
```
Comment (Agupane) [inline src/components/CardType/ThirdPartyCard.tsx:47]: same, can this be on theme?

File: src/components/CardType/OffChainCard.tsx [hunk 4]
```diff
@@ -22,18 +31,16 @@ export const OffchainCard = () => {
...
+      image={image_desktop}
+      imageSizes={
+        isMobile ? { width: 141, height: 124 } : { width: 344, height: 300 }
+      }
+      howToSVGComponent={
+        <Suspense fallback={<CircularProgress color="success" />}>
+          {isMobile ? <OffChainHowToMobile /> : <OffChainHowTo />}
+        </Suspense>
+      }
+      howToAspectRatio={isMobile ? '6/10' : '9/9'}
```
Comment (Agupane) [inline src/components/CardType/OffChainCard.tsx:43]: same

File: src/components/CardType/OffChainCard.tsx [hunk 5]
```diff
@@ -22,18 +31,16 @@ export const OffchainCard = () => {
...
-      Step 2: Prepare and submit the evidence. Each type of badge will request to accomplish some specific tasks. For example, a tweet containing your wallet address.
-      Step 3: Your submission will have to pass through Kleros's curation process. Where the community will have a few days to analyze the evidence you have submitted.
-      Step 4: Nobody has challenged your submission. The badge is minted to your address.
-      Step 4.1: Someone has challenged your submission. A Jury of Kleros will analyze your evidence and determine if it is veridic or not.
-      Step 4.2: The jury determines the evidence is veridic. The badge is minted.
-      Step 4.3: The jury determines the evidence is fraudulent. The badge is not minted.
-      `}
+      image={image_desktop}
+      imageSizes={
+        isMobile ? { width: 141, height: 124 } : { width: 344, height: 300 }
```
Comment (Agupane) [inline src/components/CardType/OffChainCard.tsx:36]: same

**Review by Agupane — CHANGES_REQUESTED** [review]

**Comment by Agupane** [comment]
In general looks much better now, good joob 👌 

This is before:
image

This is now:
image

File: src/components/CardType/index.tsx (hunk 1 above)
Comment (FedeMadoery) [reply src/components/CardType/index.tsx]: Nop, good catch

**Review by Agupane — APPROVED** [review]

**Pull request merged**